sim/dex-data.ts

```diff
@@ -664,13 +663,12 @@ export class Template extends BasicEffect implements Readonly<BasicEffect & Temp
 
 		this.fullname = `pokemon: ${data.name}`;
 		this.effectType = 'Pokemon';
-		this.speciesid = data.speciesid as ID || this.id;
-		this.species = data.species || data.name;
-		this.name = data.species;
+		this.id = data.id as ID;
+		this.name = data.name;
```

Comment:
I'm 80% sure these are no longer necessary? I think `super` covers it?

Reply:
when i tried removing them it threw a ton of errors